tedana/workflows/tedana.py

```diff
@@ -387,6 +394,10 @@ def tedana_workflow(data, tes, out_dir='.', mask=None,
     if not isinstance(gscontrol, list):
         gscontrol = [gscontrol]
 
+    # Check value of tedpca *if* it is a float
```

Comment:
Could we just call `string_or_float` here? From CLI it should get checked automatically, but the Python API won't.

Reply:
Good point!